chapter07/kds/ex6.pl

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+print "Enter some text or supply an input file as an argument:\n";
+while( <> ){
+  print "$_" if( /fred/ and /wilma/ );
```

Comment:
Perl seems to have 2 sets of logical operators for the same operations. Will be interesting to discuss this.